.prettierignore

```diff
@@ -10,3 +10,4 @@ testing/
 testing*
 *.pyc
 bin/
+.github/workflows/ci.yml
```

Comment:
why?

Reply:
It was failing in the precommit linting
Twice the key 
```
        NXF_VER:
          - "23.04.0"
          - "latest-everything"
```